app/models/event_orga.py

```diff
@@ -13,9 +15,13 @@ class EventOrgaModel(SoftDeletionModel):
     payment_currency = db.Column(db.String, nullable=False)
 
     def __init__(self,
-                 name=None):
+                 name=None,
+                 starts_at=None,
+                 payment_currency=None):
 
         self.name = name
+        self.starts_at = datetime.utcnow()
```

Comment:
If you are setting this here, then adding a constructor parameter is useless

Reply:
Also, I think adding here makes more sense.